<launch>
  <!-- The name of the object we record -->
  <arg name="model_name"/>
  <arg name="frame_nr"/>
  
  <!-- RGBD camera -->
  <include file="$(find openni2_launch)/launch/openni2.launch">
	<arg name="depth_registration" value="true" />
</include>

  <node pkg="object_recognition" type="keyboard_listener" name="keyboard_listener" output="screen">	
  </node>

  <node pkg="object_recognition" type="vision_training2d" name="vision_training2d" args="$(arg model_name) $(arg frame_nr)" output="screen">	
  </node>

</launch>
